### PR TITLE
Feat task8 memorymanagement

### PR DESCRIPTION
**Pull request by AncyJenisha**

### CODE REVIEW

File: src/Assignment8/GarbageCollection/SampleClass.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿// <copyright file="SampleClass.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace GarbageCollection
+{
+    /// <summary>
+    /// Is an empty class to create objects for it.
+    /// </summary>
+    public class SampleClass
```
Comment (krrish-soliton) [inline src/Assignment8/GarbageCollection/SampleClass.cs:10]: What is the purpose of this empty class?

File: src/Assignment8/IDisposableDemo/FileOperations.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                }
+
+                Console.WriteLine("\nContents Printed.");
+            }
+        }
+
+        /// <summary>
+        /// Disposes the file after use.
+        /// </summary>
+        public void Dispose()
```
Comment (krrish-soliton) [inline src/Assignment8/IDisposableDemo/FileOperations.cs:53]: This is not needed and IDisposable is also not needed if you don't have any disposable items.

**Review by krrish-soliton — APPROVED** [review]

File: src/Assignment8/GarbageCollection/SampleClass.cs (hunk 1 above)
Comment (AncyJenisha) [reply src/Assignment8/GarbageCollection/SampleClass.cs:10]: To create object for it and understand the work of Garbage Collector

File: src/Assignment8/IDisposableDemo/FileOperations.cs (hunk 2 above)
Comment (AncyJenisha) [reply src/Assignment8/IDisposableDemo/FileOperations.cs:53]: It was asked in the task to implement dispose method to dispose the file after use.

**Pull request merged**